scripts/populate_file_web_server.py

```diff
@@ -72,40 +74,27 @@ def __str__(self):
 
 # Helper functions
 
-def info_print(msg):
-    if PRINT_LEVEL_LUT[g_current_print_level] >= PRINT_LEVEL_LUT[PRINT_LEVEL_INFO]:
-        print(msg)
-
-def info_print_in_place(msg):
-    if PRINT_LEVEL_LUT[g_current_print_level] >= PRINT_LEVEL_LUT[PRINT_LEVEL_INFO]:
-        print(msg, end='\r')
-
-def verbose_print(msg):
-    if PRINT_LEVEL_LUT[g_current_print_level] >= PRINT_LEVEL_LUT[PRINT_LEVEL_VERBOSE]:
-        print(msg)
+def print_msg(print_level, msg, print_in_place):
```

Comment:
You can have default value for print_in_place=False. In the invocations below other than when it needs to be true, you can ignore this parameter.

Reply:
Done